lib/download-asset.js

```diff
@@ -0,0 +1,45 @@
+import fs from 'fs-extra'
+var request = require('request')
+var path = require('path')
+
+export default function downloadAsset (url, directory) {
+  return new Promise(function (resolve, reject) {
+    // build local file path from the url for the download
+    var urlParts = url.split('//')
```

Comment:
It would be nice if you are getting only the file name and append timestamp like  so : 

```js
// maybe append a uniq hash to the name
var localFile = fileNameFromUrl(url)
function fileNameFromUrl(url) {
  var matches = url.match(/\/([^\/?#]+)[^\/]*$/);
  if (matches.length > 1) {
    return matches[1];
  }
  return null;
}
```